organization/src/integrationTest/kotlin/com/cosmotech/organization/service/OrganizationServiceIntegrationTest.kt

```diff
@@ -1283,1447 +1283,1502 @@ class OrganizationServiceIntegrationTest : CsmRedisTestBase() {
+        val organizationRegistered =
+            organizationApiService.registerOrganization(createTestOrganization(name))
+        assertNotNull(organizationApiService.getOrganizationSecurity(organizationRegistered.id!!))
+      }
+    }
+
+    @Test
+    fun `getOrganizationSecurity with non existing organization`() {
+      assertThrows<CsmResourceNotFoundException> {
+        organizationApiService.getOrganizationSecurity(UNKNOWN_IDENTIFIER)
+      }
+    }
+
+    @Test
+    fun `getOrganizationSecurity with no security organization`() {
+      assertThrows<CsmResourceNotFoundException> {
+        organizationApiService.registerOrganization(createTestOrganization(name = "org1"))
+        organizationApiService.getOrganizationSecurity("org1")
+      }
+    }
+
+    @Test
+    fun `getOrganizationSecurity with not owned organization with READ_SECURITY permission`() {
+      assertDoesNotThrow {
+        val name = "o-connector-test-1"
+        runAsDifferentOrganizationUser()
+        val organizationRegistered =
+            organizationApiService.registerOrganization(
+                createTestOrganizationWithSimpleSecurity(
+                    name, TEST_USER_ID, ROLE_NONE, ROLE_VIEWER))
+        runAsPlatformAdmin()
+        assertNotNull(organizationApiService.getOrganizationSecurity(organizationRegistered.id!!))
       }
+    }
 
-      @Test
-      fun `getOrganizationSecurity with no security organization`() {
-        assertThrows<CsmResourceNotFoundException> {
-          organizationApiService.registerOrganization(createTestOrganization(name = "org1"))
-          organizationApiService.getOrganizationSecurity("org1")
-        }
+    @Test
+    fun `getOrganizationSecurity with not owned organization with no READ_SECURITY permission`() {
+      assertDoesNotThrow {
+        val name = "o-connector-test-1"
+        runAsDifferentOrganizationUser()
+        val organizationRegistered =
+            organizationApiService.registerOrganization(
+                createTestOrganizationWithSimpleSecurity(
+                    name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_NONE))
+        runAsPlatformAdmin()
+        assertNotNull(organizationApiService.getOrganizationSecurity(organizationRegistered.id!!))
       }
+    }
 
-      @Test
-      fun `getOrganizationSecurity with not owned organization with READ_SECURITY permission`() {
-        assertDoesNotThrow {
-          val name = "o-connector-test-1"
-          runAsDifferentOrganizationUser()
-          val organizationRegistered =
-              organizationApiService.registerOrganization(
-                  createTestOrganizationWithSimpleSecurity(
-                      name, TEST_USER_ID, ROLE_NONE, ROLE_VIEWER))
-          runAsPlatformAdmin()
-          assertNotNull(organizationApiService.getOrganizationSecurity(organizationRegistered.id!!))
-        }
+    @Test
+    fun `setOrganizationDefaultSecurity with owned organization and existing role`() {
+      assertDoesNotThrow {
```

Comment:
assertDoesNotThrow should be specific else it's the same as not having it

Reply:
same comment as above